Assets/Scripts/Movement.cs

```diff
@@ -162,14 +163,21 @@ private void ChargingTorch(InputAction.CallbackContext obj)
     }
     private void WhileChargingTorchThrow()
     {
```

Comment:
I noticed before that the torch would infinitely fall through the ground before. Not a big deal, but I decided to clean it up and give in a maximum distance it can move.